Objects/exceptions.c

```diff
@@ -2324,7 +2324,9 @@ AttributeError_reduce(PyAttributeErrorObject *self, PyObject *Py_UNUSED(ignored)
         return NULL;
     }
 
-    return PyTuple_Pack(3, Py_TYPE(self), self->args, state);
+    PyObject* return_value = PyTuple_Pack(3, Py_TYPE(self), self->args, state);
```

Comment:
```suggestion
    PyObject *return_value = PyTuple_Pack(3, Py_TYPE(self), self->args, state);
```
For consistency with the rest of function =)